mr/src/test/java/org/apache/iceberg/mr/TestIcebergInputFormats.java

```diff
@@ -370,7 +370,7 @@ public void testCustomCatalog() throws IOException {
     testInputFormat.create(builder.conf()).validate(expectedRecords);
   }
 
-  private abstract static class TestInputFormat<T> {
+  public abstract static class TestInputFormat<T> {
```

Comment:
Why was this changed to `public`?

Reply:
Hmm, I thought accessing `TestIcebergInputFormats.TestInputFormat.newFactory` would need it to be public. It turns out not necessary. Updated.

Reply:
I tried to minimize the changes to access modifiers in `TestIcebergInputFormat`, the last place that needs the modifier of `TestInputFormat` to be public is `getRecords` method.